app/lib/three_scale/hash_hacks.rb

```diff
@@ -17,11 +17,6 @@ def transform_values
       end
     end
 
-    def sweep
```

Comment:
What is [this](https://github.com/3scale/system/commit/c217bac039a8eb926275a755e47e2ac0c0c4fa4b) for and how do we know that it is safe to remove it?

Reply:
I looked up usages and found none.. but it doesn't belong to the PR anyways so I'll revert it